pravega/pravega
# Issue 7099: Enhance flush to storage to save epoch info.

**Change log description**  
Saving latest epoch info during flush-to-storage command execution. Each container's epoch is saved in a separate file `container_<container_ID>_epoch`. This is required during restore from lts.

**Purpose of the change**  
Fixes #7099

**What the code does**  
Saves epoch info after executing flush-to-storage.

**How to verify it**  
On admin cli execute flush-to-storage command and verify creation/existence of files named in format `container_<container_ID>_epoch` Eg- `container_0_epoch`.
